app/models/solidus_bolt/gateway.rb

```diff
@@ -39,6 +59,15 @@ def purchase(_float_amount, _response_code, _gateway_options)
 
     private
 
+    def can_capture?(reference, payment_method)
```

Comment:
I found that code in the [solidus codebase](https://github.com/solidusio/solidus/blob/e878076f2ed670d07654ab6293a16588743f2fa6/core/app/models/spree/payment_source.rb#L22), maybe we don't have to implement that method. WDYT?

Reply:
Removed this method now
[21d5259](https://github.com/nebulab/solidus_bolt/commit/21d52591df40a22f4b3b28704bab3c75c9e6d2e1)